export-helpers/README.md

```diff
@@ -0,0 +1,28 @@
+# export-helpers
+
+## What it does
+
+This extension demonstrates how to use export helpers in Firefox to share
+JavaScript objects defined in content scripts with scripts loaded by web pages.
+
+It has a background script that does two things:
+
+* execute a content script in the current tab, when the user clicks a browser action
+* listen for messages from the content script, and display a notification when the message arrives.
+ 
+The content script then:
+ 
+ * defines a function `notify()` and exports that to the page as a property
```

Comment:
Another use case of `cloneInto` (without the `cloneFunctions` option) that could be reasonable to provide in this example is the following:
- a content script has received from the messaging API (e.g. onMessage or a Port) and wants to forward it to the page into a custom event:
  - on Chrome the object properties are accessible to the webpage javascript code without doing anything
  - on Firefox the object properties are not accessible to the webpage javascript and the objects needs to be explicitly cloned using cloneInto before forwarding as the details of the custom event sent to the page.

The following commit shows a similar change applied to the WebVR Extension to make it to work correctly when running as a WebExtension on Firefox: 

- https://github.com/rpl/WebVR-Extension/commit/1548d4e40a2450fcd13ae8804415970bf9f7bd75